docs/management/advanced-options.asciidoc

```diff
@@ -47,6 +47,7 @@ increase request processing time.
 `histogram:barTarget`:: When date histograms use the `auto` interval, Kibana attempts to generate this number of bars.
 `histogram:maxBars`:: Date histograms are not generated with more bars than the value of this property, scaling values
 when necessary.
+`visualize:enableLabs`:: Enables experimental visualizations. Default is `true`. If this is turned off, users will not be able to create visualizations that are considered experimental. They will also not be able to add these visualizations to the dashboard. 
```

Comment:
Could we maybe change the last sentence to "They will also not be able to add or view these visualizations on a dashboard."

The way it's currently written it sounds a bit like any visualization already in use, will still work fine.